app/controllers/email_alert_subscriptions_controller.rb

```diff
@@ -19,13 +18,25 @@ def create
 private
 
   def content
-    @content ||= Services.content_store.content_item(request.path)
+    @content ||= fetch_content
```

Comment:
We should use the Finder API class so that we're not duplicating how to fetch the finder content item and handle development content items.

```suggestion
    @content ||= FinderApi.new(request.path, nil).content_item
```